### PR TITLE
add styles

### PR DESCRIPTION
**Pull request by merzavko**
added common styles for project pages
added needed images

### CODE REVIEW

**Comment by okrasn** [comment]
Please rebase

**Comment by merzavko** [comment]
done

File: app/styles/scss/modules/_template.scss [hunk 1]
```diff
@@ -49,8 +49,45 @@ a {
...
+    text-decoration: none;
+    color: $headerTextColor;
+    font-weight: bold;
+}
+.navbar-default .navbar-brand {
+    color: $aticleTitleColor;
+    font-weight: bold;
+    text-decoration: none;
+}
+.navbar-default .navbar-nav > li > a {
```
Comment (okrasn) [inline app/styles/scss/modules/_template.scss]: Please use class for a tag

File: app/styles/scss/modules/_template.scss [hunk 2]
```diff
@@ -49,8 +49,45 @@ a {
...
+.pagination .active a, .pagination .active a:hover{
     background-color: #eee;
     color: #2d6594;
     border: 1px solid #ddd;
+}
+.navbar-default {
+    background-color: #fff;
+    border-color: #fff;
+}
+.navbar-default .navbar-nav > .active > a, .navbar-default .navbar-nav > .active > a:focus, .navbar-default .navbar-nav > .active > a:hover {
```
Comment (okrasn) [inline app/styles/scss/modules/_template.scss]: Please build nested classes

File: app/styles/scss/modules/_template.scss [hunk 3]
```diff
@@ -49,8 +49,45 @@ a {
     color: #5b8bac;
     text-decoration: underline;
 }
-.pagination .active a{
+.pagination .active a, .pagination .active a:hover{
```
Comment (okrasn) [inline app/styles/scss/modules/_template.scss]: Please not put styles on tag, use class or >

File: app/styles/scss/modules/_course-block.scss [hunk 4]
```diff
@@ -74,6 +72,7 @@
         }
     }
 
+<<<<<<< HEAD
```
Comment (okrasn) [inline app/styles/scss/modules/_course-block.scss]: Please resolve conflicts

**Pull request merged**